lib/Ed25519VerificationKey2020.js

```diff
@@ -83,7 +83,8 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
    */
   static async from(options) {
     if(options.type === 'Ed25519VerificationKey2018') {
-      return Ed25519VerificationKey2020.fromEd25519VerificationKey2018(options);
+      return Ed25519VerificationKey2020
+        .fromEd25519VerificationKey2018({keyPair: options});
```

Comment:
Hmm, I'm guessing this was just broken before and we didn't have a test for it?

Reply:
Yeah, sadly :( 
(It's not disastrous, in that, DB is not currently using that, but this adds a fix & tests it).